setup.cfg

```diff
@@ -46,7 +46,7 @@ python_requires = >= 3.9
 
 [options.extras_require]
 dev =
-    ghga-service-chassis-lib[dev]==0.17.0
+    ghga-service-chassis-lib[dev]==0.17.7
 all =
     %(dev)s
```

Comment:
```suggestion
```

You can probably remove that as most of the stuff should be in `requirements-dev-common.txt` now.